### PR TITLE
Shuffleboard Debug Mode vs Non Debug Mode

### PR DESCRIPTION
**Pull request by stewluj**
Logs more or less depending whether or not debug mode is true

### CODE REVIEW

File: src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java [hunk 1]
```diff
@@ -241,17 +174,57 @@ public boolean getSensorOverride() {
...
+			shuffleboardTab
+					.addDouble("Right Intake Motor Temp", () -> intakeMotorRight.getMotorTemperature())
+					.withSize(1, 1)
+					.withWidget(BuiltInWidgets.kTextView);
+			shuffleboardTab
+					.addDouble("Ingest Motor Temp", () -> ingestMotor.getMotorTemperature())
+					.withSize(1, 1)
+					.withWidget(BuiltInWidgets.kTextView);
+			shuffleboardTab
+					.addDouble("Index Motor Temp", () -> feederMotor.getMotorTemperature())
```
Comment (TAKBS2412) [inline src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java]: This one is named Index motor temp, but uses the feeder motor...

File: src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java [hunk 2]
```diff
@@ -231,13 +228,24 @@ public void setAngleSpeed(double Speed) {
...
+					.add(new SparkPIDWidget(launcherTopPIDController, "launcherTopPID"))
+					.withPosition(0, 0);
+			Shuffleboard.getTab("Launcher")
+					.add(new SparkPIDWidget(launcherBottomPIDController, "launcherBottomPID"))
+					.withPosition(1, 0);
+
+			Shuffleboard.getTab("Launcher")
+					.addDouble("Bottom FlyWheel Temp", () -> launcherBottomMotor.getMotorTemperature());
+			Shuffleboard.getTab("Launcher")
+					.addDouble("Top FlyWheel Temp", () -> launcherBottomMotor.getMotorTemperature());
```
Comment (bobtseattle) [inline src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java]: Labeled as top but displays bottom motor.

**Review by bobtseattle — DISMISSED** [review]
Noting a mismatch between the label and data.

**Review by TAKBS2412 — DISMISSED** [review]

**Comment by TAKBS2412** [comment]
Approved (pending formatting issues).

**Review by bobtseattle — APPROVED** [review]
LGTM

File: src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java [hunk 3]
```diff
@@ -269,13 +266,22 @@ public void manualSetpoint(double setpoint) {
...
+					.add(new SparkPIDWidget(launcherTopPIDController, "launcherTopPID"))
+					.withPosition(0, 0);
+			Shuffleboard.getTab("Launcher")
+					.add(new SparkPIDWidget(launcherBottomPIDController, "launcherBottomPID"))
+					.withPosition(1, 0);
+
+			Shuffleboard.getTab("Launcher")
+					.addDouble("Bottom FlyWheel Temp", () -> launcherBottomMotor.getMotorTemperature());
+			Shuffleboard.getTab("Launcher")
+					.addDouble("Top FlyWheel Temp", () -> launcherTopMotor.getMotorTemperature());
```
Comment (KangarooKoala) [inline src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java:283]: Have we tested the placement of these? Bottom previously had `.withPosition(0, 3)` and `.withSize(2, 1)` and top previously had `.withPosition(2, 3)` and `.withSize(2, 1)`, but without those I'm worried they will be assigned a spot that overlaps with something else.

**Pull request merged**